Automattic/pocket-casts-android
# Add up next icon to episode chip

## Description
This adds an up-next icon to the `EpisodeChip`.

In addition, this updates the episode chip to vertically center the podcast image when there are no icons present. Unfortunately, you can't test this currently because the only screens using this chip right now are the Up Next screen and the Downloads screen and I certainly hope that any podcast on one of those screens should either be in the Up Next or downloaded. 😄 I've included a screenshot below though so you can see what it would look like.

> **Note**
> This PR builds on https://github.com/Automattic/pocket-casts-android/pull/909.

## Testing Instructions
1. Go to the Up Next queue or the Downloads page
2. Observe that the up-next icon and download icons are presented when appropriate

## Screenshots or Screencast 

| Both icons | Up Next only | Downloaded only | No icons |
| --- | --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/4656348/234697996-3b63eeae-5f9f-4c58-b645-5a80b62e1586.png) | ![image](https://user-images.githubusercontent.com/4656348/234698045-32318683-265b-44a9-840f-86ad3fd6b439.png) | ![image](https://user-images.githubusercontent.com/4656348/234698159-c0903e68-917a-4116-9b85-1b6dd3fadfdb.png) | ![image](https://user-images.githubusercontent.com/4656348/234698661-4165d348-7278-40dd-9bbf-1299a8b4e2fa.png) |

## Checklist
